components/stack/stack_constants.js

```diff
@@ -0,0 +1,18 @@
+export const DT_STACK_DIR = {
+  default: 'column',
+  column: 'column',
+  row: 'row',
+  'row-reverse': 'row-reverse',
+  'column-reverse': 'column-reverse',
+};
+
+export const DT_STACK_RESPONSIVE_BREAKPOINTS = ['sm', 'md', 'lg', 'xl'];
+
+export const DT_STACK_GAP = {
+  100: '100',
```

Comment:
I didnt put 0, because we dont have any validator here also 0 is the default value. I didnt see it necessary.

But now we will have validator, so your POV its correct :)